enrichment_logbook_uploader/logbook_upload.py

```diff
@@ -189,7 +217,7 @@ def main(activity_enrichment_cookies=None):
 
             raise SystemExit(1)
 
-    payloads = generate_payload(df, logbook_header_id)
+    payloads = generate_payload(df, logbook_header_id, headers)
```

Comment:
You should define a new function for this feature, and call it after sending the initial payloads
```suggestion
    payloads = generate_payload(df, logbook_header_id)
```